Add include algorithm

This fixes an error when building ignition-gazebo:

    error: 'find' is not a member of 'std'; did you mean 'bind'?

Upstream-Status: Pending

Signed-off-by: Rob Woolley <rob.woolley@windriver.com>
diff --git a/include/gz/physics/detail/InspectFeatures.hh b/include/gz/physics/detail/InspectFeatures.hh
index 7a8b6dbc..bf5f5459 100644
--- a/include/gz/physics/detail/InspectFeatures.hh
+++ b/include/gz/physics/detail/InspectFeatures.hh
@@ -18,6 +18,7 @@
 #ifndef IGNITION_PHYSICS_DETAIL_INSPECTFEATURES_HH
 #define IGNITION_PHYSICS_DETAIL_INSPECTFEATURES_HH
 
+#include <algorithm>
 #include <set>
 #include <string>
 #include <tuple>
